File: src/lib/ir/__fixtures__/cpp/14-switch-fallthrough.cpp
int describe(int v) {
    int r = 0;
    switch (v) {
        case 1:
            r = 1;
        case 2:
            r = 2;
            break;
        case 3:
            r = 3;
            break;
        default:
            r = 9;
    }
    return r;
}

File: src/lib/ir/__fixtures__/js/14-switch-fallthrough.js
function describe(x) {
  let out = '';
  switch (x) {
    case 0:
    case 1:
      out = 'low';
      break;
    case 2:
      out = 'two';
      break;
    default:
      out = 'high';
  }
  return out;
}
